src/components/TerminalOutput/TerminalOutput.js

```diff
@@ -45,32 +52,80 @@ class TerminalOutput extends PureComponent<Props> {
     this.node.scrollTop = maxScrollTop > 0 ? maxScrollTop : 0;
   };
 
+  handleClear = () => {
+    const { task, clearConsole } = this.props;
+
+    if (!task) {
+      return;
+    }
+
+    clearConsole(task);
+  };
+
   render() {
-    const { width, height, logs } = this.props;
+    const { width, height, title, task } = this.props;
+
+    // TODO: Check if `task` is actually always defined.
+    const logs = task ? task.logs : [];
+
+    console.log(task);
```

Comment:
Nit: debugging log?

Reply:
Ah! I should also do that `TODO`, meant to look at that before opening a PR. Thanks for catching.